pkg/api/models/approval.go

```diff
@@ -18,3 +20,21 @@ type Approval struct {
 	// Time of the event
 	Time string `json:"time,omitempty"`
 }
+
+// ToJSON converts object to JSON string
+func (a *Approval) ToJSON() ([]byte, error) {
+	if a == nil {
```

Comment:
I believe this check can be removed - if the `ToJSON()` function is attempted to be called on a nil value, it will not reach this check anyway

Reply:
good catch, fixed :)